caravel/controllers/forms.py

```diff
@@ -33,11 +33,11 @@ class ListingForm(flask_wtf.Form):
     
 
 class NewListingForm(ListingForm):
-    principal = PrincipalField("Seller",
-        validators=[DataRequired(), OnlyValid()])
+    principal = PrincipalField("Seller", validators=[DataRequired()])
```

Comment:
Maa-ybe. The problem is we don't know if Shibboleth fails. (As far as I can tell.) I think they just a "LOGIN INCORRECT" message or something equivalently unhelpful.